lib/config.js

```diff
@@ -122,6 +123,7 @@ var ENV_TABLE = {
   disableInstrumentations: 'ELASTIC_APM_DISABLE_INSTRUMENTATIONS',
   disableSend: 'ELASTIC_APM_DISABLE_SEND',
   environment: 'ELASTIC_APM_ENVIRONMENT',
+  exitSpanMinDuration: 'ELASTIC_APM_EXIT_SPAN_MIN_DURATION',
```

Comment:
The spec says this is a config var that could come from central config, so an entry should be added to `CENTRAL_CONFIG` and an addition made to test/central-config-enabled.test.js.

Reply:
✅